mycore-webtools/src/main/java/org/mycore/webtools/vue/MCRVueRootServlet.java

```diff
@@ -132,26 +133,30 @@ public MCRContent getContent(HttpServletRequest req, HttpServletResponse resp) t
         String pathInfo = req.getPathInfo();
         String indexHtmlPage = getIndexPage();
         String indexHtmlPath = req.getServletPath() + "/" + indexHtmlPage;
-        URL resource = getServletContext().getResource(req.getServletPath() + pathInfo);
 
-        if (resource != null && !pathInfo.endsWith("/") && !pathInfo.endsWith(indexHtmlPage)) {
-            return new MCRURLContent(resource);
-        } else {
-            URL indexResource = getServletContext().getResource(indexHtmlPath);
-            org.jdom2.Document mycoreWebpage = getIndexDocument(indexResource, getAbsoluteServletPath(req));
-            if (pathInfo != null && pathInfo.endsWith("/404")) {
-                /* if there is a requested route which does not exist, the app should
-                 * redirect to this /404 route the get the actual 404 Code.
-                 * see also https://www.youtube.com/watch?v=vjj8B4sq0UI&t=1815s
-                 * */
-                resp.setStatus(HttpServletResponse.SC_NOT_FOUND);
-            }
-            try {
-                return getLayoutService().getTransformedContent(req, resp, new MCRJDOMContent(mycoreWebpage));
-            } catch (TransformerException | SAXException e) {
-                throw new IOException(e);
+        if (pathInfo != null && !pathInfo.endsWith("/") && !pathInfo.endsWith(indexHtmlPage)) {
+            URL resource = MCRResourceHelper.getResourceUrl(req.getServletPath() + pathInfo);
+            if (resource != null) {
+                return new MCRURLContent(resource);
             }
         }
+
+        URL indexResource = MCRResourceHelper.getResourceUrl(indexHtmlPath);
```

Comment:
See above, `indexResource` is `null`.

Reply:
I'll look into it. Can you tell me the easiest way to test this?

Reply:
I test in MIR with the job queue manager via `/jobqueue`. I already suspected the paths with the starting `/`, but that doesn't solve the problem either.